pkg/cloudprovider/aws/launchtemplate.go

```diff
@@ -149,21 +150,22 @@ func (p *LaunchTemplateProvider) ensureLaunchTemplate(ctx context.Context, optio
 }
 
 func (p *LaunchTemplateProvider) createLaunchTemplate(ctx context.Context, options *launchTemplateOptions) (*ec2.LaunchTemplate, error) {
+	clusterName := ptr.StringOrEmpty(options.Cluster.Name)
```

Comment:
consider just inlining ptr.StringValue(options.Cluster.Name) to simplify the code (i.e. decrease the number of variables/concepts by 1)